resources/public/api/conf/common/overview.md

```diff
@@ -1,17 +1,11 @@
-This API allows developers to:
+Use the Property Business API to submit, edit and retrieve information about a customer’s UK and foreign property income.
 
-- list, retrieve, create or amend an individual’s UK Property Income & Expenses Period Summaries
-- retrieve, create, amend or delete an individual’s UK Property Annual Submissions
-- list, retrieve, create or amend an individual’s Foreign Property Income & Expenses Period Summaries
-- retrieve, create, amend or delete an individual’s Foreign Property Annual Submissions
-- retrieve, create, amend or delete an individual’s Historic FHL and Non-FHL UK Property Business Annual Submissions
-- list, retrieve, create or amend an individual’s Historic FHL and Non-FHL UK Property Business Income & Expenses Period Summaries
+This API is related to the following customer journeys:
 
-The "historic" endpoints are for data relating to tax years from 2017-18 to 2021-22. 
-For later years, use the standard endpoints.
+Submit business income and expenses
+Submit quarterly updates
+Submit annual updates 
```

Comment:
Can we make these bullet points please? You can add * at the start of the line to make it a bullet point. Then leave a blank line after the last bullet point